config/webpack.config.dev.js

```diff
@@ -168,6 +169,133 @@ module.exports = {
     new HtmlWebpackPlugin({
       inject: true,
       template: paths.appHtml,
+      cspPlugin: { //adds CSP Meta tag plugin
+        enabled: true,
+        policy: {
+          'default-src': "'none'",
+          'frame-ancestors': "'self'",
+          'manifest-src': "'self'",
+          'base-uri': "'self'",
+          'style-src': [
+            "'self'",
+            "'unsafe-inline'",
+            "blob:",
+            "https://fonts.googleapis.com",
+            "https://*.netlify.com"
+          ],
```

Comment:
I understand you needed to add netlify  as a source to prevent errors. Would it be better to specify that it's `giftaid-react.netlify.com` we allow things from instead of any old netlify hosted source? I guess you can't easily use the asterisk there though.. 
Same goes for *.google.com and *.cloudfront.net. Not sure if this is still a security problem...

Reply:
The netlify issue is only relevant in the preview environment. 
I also have no idea how netlify could host malicious scripts. 
A malicious actor could probably try to create a domain similar to `netlify.com` say `netllify.com` and it wouldn't be `https:`
We can't be that specific with `cloudfront.net` as sub domains are unique. 
The security problem is that we currently have no Security headers. 
This PR doesn't aim to solve all the security issues. We still need to set `X-XSS-Protection` and `X-Content-Type-Options` in the response headers from Cloudfront and add unique hashes to all inline scripts

Reply:
I see